src/chains/tests/test_views.py

```diff
@@ -295,6 +301,29 @@ def test_null_ens_registry_address(self) -> None:
         self.assertEqual(response.json()["ensRegistryAddress"], None)
 
 
+class ChainLogoTests(APITestCase):
```

Comment:
These tests should be moved to `src/chains/tests/test_models.py` as we are not testing the view directly here.

Also extending `TestCase` should be enough for these tests.

Reply:
I moved it in 82083887dcd88a0e34c9b70fe269a468686c90e2, along with that of the `currency_logo` for consistency.